_data/themes/beach.yml

```diff
@@ -0,0 +1,47 @@
+# should we name these colors based on what elements they change or keep them the same name as the uswds variable colors?
+
+colors:
+# content link color, hover color in nav, active highlight in nav, primary button bg color, return to to link color
+# usa-banner link color ???
+  primary: 'red'
+# content link hover color, usa-banner link hover color, primary button hover bg color, return to top link hover color
+  primary-darker: 'pink'
+# hero bg color, content section bg color
+  primary-darkest: 'maroon'
+  # primary-darker: 'darken($color-primary, 50%)'
+  # primary-darkest: 'darken($color-primary, 100%)'
+# hero alt heading color, hero link color, content section heading color
+  primary-alt: 'lightblue'
+  primary-alt-dark: 'purple'
+  primary-alt-darkest: 'blue'
+  # primary-alt-dark: 'darken($color-primary-alt, 5%)'
+  # primary-alt-darkest: 'darken($color-primary-alt, 10%)'
+  primary-alt-light: 'lighten($color-primary-alt, 5%)'
+  primary-alt-lightest: 'lighten($color-primary-alt, 10%)'
+# dont see any changes for these secondary items?
+  secondary: 'DarkOliveGreen'
+  secondary-dark: 'Olive'
+  secondary-darkest: 'OliveDrab'
+  # secondary-dark: 'darken($color-secondary, 5%)'
+  # secondary-darkest: 'darken($color-secondary, 10%)'
+  secondary-light: 'lighten($color-secondary, 5%)'
+  secondary-lightest: 'lighten($color-secondary, 10%)'
+# header site title, active top nav link color, left nav link color, link color in primary footer and main footer
+# text in usa-banner ???
+  base: 'SaddleBrown'
+# dont see any changes what does this change?
+  focus: 'lime'
+# visited link color, return to top visited color
+  visited: 'Cyan'
+# changes primary footer and the usa-banner
+  primary-footer-bg: 'orange'
+  footer-bg: 'yellow'
+  left-nav-hover-bg: 'MediumAquamarine'
+
```

Comment:
There were a few other colors I thought should be in the theme for the footer and left nav